OrbitGl/App.cpp

```diff
@@ -1011,8 +999,8 @@ void OrbitApp::LoadModules() {
 
 //-----------------------------------------------------------------------------
 void OrbitApp::LoadRemoteModules() {
-  GetSymbolsManager()->LoadSymbols(m_ModulesToLoad,
-                                           Capture::GTargetProcess);
+  GetSymbolsManager()->LoadSymbols(m_ModulesToLoad, Capture::GTargetProcess,
+                                   nullptr /*session*/);
```

Comment:
Why can a session be nullptr? Isn't there an overload without session?

Reply:
Session can be null if we don't have a session to apply to the loaded symbols.  I made the default param nullptr so that the user doesn't need to pass it in if there's none.